Disable GL_EXT_debug_marker due to crashes on x86 with Mesa drivers.

diff --git gfx/webrender/src/query.rs gfx/webrender/src/query.rs
index 999abc749115..68850ef025e4 100644
--- gfx/webrender/src/query.rs
+++ gfx/webrender/src/query.rs
@@ -274,17 +274,20 @@ pub struct GpuMarker {
 
 impl GpuMarker {
     fn new(gl: &Rc<gl::Gl>, message: &str) -> Self {
+#[cfg(not(target_arch = "x86"))]
         gl.push_group_marker_ext(message);
         GpuMarker { gl: Rc::clone(gl) }
     }
 
     fn fire(gl: &Rc<gl::Gl>, message: &str) {
+#[cfg(not(target_arch = "x86"))]
         gl.insert_event_marker_ext(message);
     }
 }
 
 impl Drop for GpuMarker {
     fn drop(&mut self) {
+#[cfg(not(target_arch = "x86"))]
         self.gl.pop_group_marker_ext();
     }
 }
